packages/markdown/components/Code/index.jsx

```diff
@@ -5,12 +5,32 @@ const PropTypes = require('prop-types');
 // apps. Necessary because of people like this:
 // https://github.com/codemirror/CodeMirror/issues/3701#issuecomment-164904534
 const syntaxHighlighter = typeof window !== 'undefined' ? require('@readme/syntax-highlighter') : false;
+const copy = require('copy-to-clipboard');
+
+const remapLang = {
+  c: 'cplusplus',
+  'c++': 'cplusplus',
+  cpp: 'cplusplus',
+  docker: 'dockerfile',
+  html: 'htmlmixed',
+  js: 'javascript',
+  py: 'python',
+  sh: 'shell',
+  bash: 'shell',
+  mysql: 'sql',
+};
 
 function Code(props) {
-  const { className, children, lang, meta } = props;
-  const language = (className || '').replace('language-', '');
+  const { children, lang, meta } = props;
+  const language = lang in remapLang ? remapLang[lang] : lang;
+  const classes = ['rdmd-code', `lang-${language}`];
   return (
-    <code className={language ? `lang-${language}` : null} data-lang={lang} name={meta}>
+    <code className={classes.join(' ')} data-lang={language} name={meta}>
+      {!syntaxHighlighter || (
+        <button onClick={() => copy(children[0])}>
+          <i className="fa fa-copy"></i>
+        </button>
+      )}
```

Comment:
Not sure if this needs to be wrapped in the `!syntaxHighlighter` conditional, but tbh I didn't want to find out if the copy dependency was going to break our SSR #bettersafethansorry?